lambdas/services/create_document_reference_service.py

```diff
@@ -30,7 +30,8 @@
 
 class CreateDocumentReferenceService:
     def __init__(self):
-        self.s3_service = S3Service()
+        create_document_aws_role_arn = os.getenv("PRE_SIGN_ASSUME_ROLE")
```

Comment:
probably better to add this new env var to the handler's decorator, so that in case this is missing we are notified